my_project/global_scheme.py

```diff
@@ -768,14 +768,14 @@
         "name": "Absolute humidity",
         "color": ["#ffe600", "#00c8ff", "#0000ff"],
         "si": {
-            "unit": "kg water/m<sup>3</sup> dry air",
-            "range": [0, 0.03],
+            "unit": "g water/kg dry air",
```

Comment:
The unit is changed as mess/mess, and it also uses g/kg to make the data bigger, and it will be easier to read.

Reply:
Why have you removed the range? Is there a reason why it is not longer needed?